modules/sso.php

```diff
@@ -219,8 +219,10 @@ public function register_settings() {
 	 * @since 2.7
 	 **/
 	public function render_require_two_step() {
+		$require_two_step = 1 == apply_filters( 'jetpack_sso_require_two_step', get_option( 'jetpack_sso_require_two_step' ) );
```

Comment:
Could you add inline doc here, mentioning that the filter is documented in modules/sso.php?

Thanks!